api/v01/entities/vrp_input.rb

```diff
@@ -279,7 +280,9 @@ module VrpMisc
     optional(:router_mode, type: String, desc: '`car`, `truck`, `bicycle`, etc... See the Router Wrapper API doc')
     optional(:router_dimension, type: String, values: ['time', 'distance'], desc: 'time or dimension, choose between a matrix based on minimal route duration or on minimal route distance')
     optional(:speed_multiplier, type: Float, default: 1.0, desc: 'multiply the current modality speed, default : 1.0')
-    optional(:skills, type: Array[String], desc: 'Particular abilities required by a vehicle to perform this subtour', coerce_with: ->(val) { val.is_a?(String) ? val.split(/,/) : val })
+    optional(:skills, type: Array[Symbol],
```

Comment:
Have you looked what swagger displays for symbol type ?

Reply:
I am not sure how to access swagger from my local code.

Reply:
![imagen](https://user-images.githubusercontent.com/28341516/112964864-fe985100-9148-11eb-9301-a6812299f135.png)

Reply:
And on the up to date swagger-ui ?

Reply:
This already is the up to date swagger-ui. I opened the file with this code running on server/worker.